### ISSUE TITLE
[Módulo 05] - [Automação de Testes] - Questionário

### ISSUE THREAD

**Issue by RebecaPR** [cwi-reset/edicao-04-level-1#332]
Boa tarde gente!

Se eu não me engano acho que havia um formulário no módulo 5 sobre Cypress, mas não encontrei. Alguém sabe se ele foi retirado?

Obrigada!

**Comment by taahferreira** [comment]
@RebecaPR achei esse link no final da pagina da segunda aula, isso que tu procuravas ? 
https://cwi.typeform.com/to/wjxj4wM9?typeform-source=cwi-reset.github.io

**Comment by RebecaPR** [comment]
Sim...!! eu fui ver no material do reset... quando eu copiei o material pra minha página não aparece o questionário não sei pq... rsrsr... Muito Obrigada!

**Comment by taahferreira** [comment]
@RebecaPR  de nada !  :)
